docs/src/main/tut/datatypes/either.md

```diff
@@ -212,132 +212,6 @@ magic("123") match {
 }
 ```
 
-## Either in the small, Either in the large
-Once you start using `Either` for all your error-handling, you may quickly run into an issue where
```

Comment:
This section talks about using local ADTs for local errors and wrapping as you go wider and wider in your application. Variance is tangentially relevant to Solution 1 but does not help with Solution 2.